### PR TITLE
DOCS-10905: Anypoint DataGraph commands

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Review by sbehrends — DISMISSED** [review]
LGTM 🚀

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 1]
```diff
@@ -313,8 +324,10 @@ This command does not take any options, except for the default ones: `--help`, `
 > account business-group list [options]
 ----
 
-This command displays all xref:access-management::organization.adoc#business-groups[business groups]. It returns the name of the business group, the type ('Master' or 'Business unit') and the Id. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command displays all xref:access-management::organization.adoc#business-groups[business groups]. It returns the name of the business group, the type ('Master' or 'Business unit') and the Id.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc]:
```suggestion
This command displays all xref:access-management::organization.adoc#business-groups[business groups]. It returns the name of the business group, the type ('Master' or 'Business unit'), and the ID.
```

Comment (dukesphere) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc]: See style guide on "ID": https://confluence.internal.salesforce.com/display/MTDT/Term+Presentation+Reference#TermPresentationReference-I

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 2]
```diff
@@ -1002,6 +1031,38 @@ Besides the default `--help`, `-f`/`--fields` and `-o`/`--output` options, this
 |`--sort` | Sorts the results in the field name passed | `api-mgr tier list --sort "Latest Version"`
 |===
 
+=== datagraph load-balancer-config add
+
+----
+> datagraph load-balancer-config add <dlbUrl>
+----
+This command adds a dedicated load balancer configuration specified by `<dlbUrl>` to Anypoint DataGraph.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1039]: Does dlbUrl mean the URL of the dedicated load balancer? Description says "a dedicated load balancer configuration," which sounds more general. Maybe people can add things other than a URL?

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 3]
```diff
@@ -1002,6 +1031,38 @@ Besides the default `--help`, `-f`/`--fields` and `-o`/`--output` options, this
...
+// CLI DEFAULTS
+include::partial$cli-default-options.adoc[tag=CLIdefaultOptions]
+
+=== datagraph load-balancer-config describe
+
+----
+> datagraph load-balancer-config describe [options]
+----
+
+This command displays a dedicated load balancer URL for Anypoint DataGraph.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1052]: this one talk about the dlbURL. Was there possibly a mixup with the previous?

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 4]
```diff
@@ -1318,8 +1387,10 @@ This command does not take any options, except for the default ones: `--help`, `
 This command deprecates the asset passed in `<assetIdentifier>`.
 
 Argument `assetIdentifier` should be formatted as follows: `([group_id]/)<asset_id>/<version>`. +
-If `group_id` is not specified, it defaults to the currently selected Organization ID. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+If `group_id` is not specified, it defaults to the currently selected Organization ID.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1392]: Guess it's obvious enough here (and below) that "it" means the value of the group ID setting.

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 5]
```diff
@@ -1361,16 +1437,19 @@ This command does not take any options, except for the default ones: `--help`, `
...
 
 === runtime-mgr cloudhub-application list
 
 ----
 > runtime-mgr cloudhub-application list [options]
 ----
 
-This command lists all applications available in your Anypoint Platform CLI. It returns your application name, its status, the number of vCores assigned and the last time it was updated. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command lists all applications available in your Anypoint Platform CLI. It returns your application name, its status, the number of vCores assigned and the last time it was updated.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1451]: do we use Oxford comma here? "... assigned, and the..."

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 6]
```diff
@@ -1380,18 +1459,23 @@ This command does not take any options, except for the default ones: `--help`, `
 
 This command displays information on the application you pass in `<name>`. +
 You can start typing your application's name and press `tab` for Anypoint Platform CLI to autocomplete it, or you can double tap `tab` for a full list of all the values you can pass. +
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1463]: Is "double tab" hyphenated? "double-tap"?

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 7]
```diff
@@ -1380,18 +1459,23 @@ This command does not take any options, except for the default ones: `--help`, `
...
-It will return data such as the application's domain, its status, last time it was updated, the Mule version, the ZIP file name, the region, monitoring, and workers; as well as `TRUE` or `FALSE` information for persistent queues and static IPs enablement. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+It will return data such as the application's domain, its status, last time it was updated, the Mule version, the ZIP file name, the region, monitoring, and workers; as well as `TRUE` or `FALSE` information for persistent queues and static IPs enablement.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc]: I'm not sure about "data such as ... monitoring, and workers; as well as ...".

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 8]
```diff
@@ -1380,18 +1459,23 @@ This command does not take any options, except for the default ones: `--help`, `
...
 ----
 > runtime-mgr cloudhub-application describe-json  [options] <name>
 ----
 
-This command returns the raw JSON response of the application you specify in `<name>`. +
-You can start typing your application's name and press `tab` for Anypoint Platform CLI to autocomplete it, or you can double tap `tab` for a full list of all the values you can pass. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command returns the raw JSON response of the application you specify in `<name>`. 
+
+You can start typing your application's name and press `tab` for Anypoint Platform CLI to autocomplete it, or you can double tap `tab` for a full list of all the values you can pass.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1477]: same Q re "double-tap" and all instances below. 

Also, could this be an include?

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 9]
```diff
@@ -1560,9 +1657,12 @@ This command does not take any options, except for the default ones: `--help`, `
 > runtime-mgr cloudhub-application copy [options] <source> <target>
 ----
 
-This command copies the CloudHub application passed in `source` to the target passed in `target`. +
-Arguments `source` and `target` should be formatted as follows: `([group_id]/)<asset_id>/<version>`. +
-If `group_id` is not specified, it defaults to the currently selected Organization ID. +
+This command copies the CloudHub application passed in `source` to the target passed in `target`. 
+
+Arguments `source` and `target` should be formatted as follows: `([group_id]/)<asset_id>/<version>`.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1664]: Is "should be" verboten?

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 10]
```diff
@@ -1615,20 +1716,26 @@ xref:runtime-fabric::install-rtfctl.adoc#supported-commands[Supported rtfctl Com
 > runtime-mgr standalone-application artifact [options] <identifier> <directory>
 ----
 
-This command downloads the application artifact of the `identifier` application, to the directory passed in `directory`. +
-The `identifier` parameter can be either an application ID or name. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command downloads the application artifact of the `identifier` application, to the directory passed in `directory`.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1721]: Is the comma needed "application, to"?

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 11]
```diff
@@ -1638,8 +1745,10 @@ To redeploy an app and set a logging level or turn on Insight event tracking, se
 > runtime-mgr standalone-application describe-json [options] <identifier>
 ----
 
-This command describes the standalone (on-premises) application passed in `identifier` as a raw JSON response. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command describes the standalone (on-premises) application passed in `identifier` as a raw JSON response.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1750]: What do we mean by a standalone application? We have standalone instances of Mule. Do they mean applications deployed on-premises?

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 12]
```diff
@@ -1674,17 +1783,21 @@ To set multiple logging levels, provide multiple `--log` options, for example, `
 > runtime-mgr standalone-application start [options] <identifier>
 ----
 
-This command starts the standalone (on-premises) application passed in `identifier`. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command starts the standalone (on-premises) application passed in `identifier`.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1788]: Same Q re standalone app. Same goes for remaining instances of this.

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 13]
```diff
@@ -1722,31 +1837,37 @@ Besides the default `--help`, `-f`/`--fields` and `-o`/`--output` options, this
...
+include::partial$cli-default-options.adoc[tag=CLIdefaultOptions]
 
 === runtime-mgr standalone-application copy
 
 ----
 > runtime-mgr standalone-application copy [options] <source> <target> <targetIdentifier>
 ----
 
-This command copies the standalone (on-premises) application passed in `source` to the target passed in `target` and the server, server group or cluster ID or Name passed in `targetIdentifier`. +
+This command copies the standalone (on-premises) application passed in `source` to the target passed in `target` and the server, server group or cluster ID or Name passed in `targetIdentifier`.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1864]: Should it be "... server group or cluster ID, or Name passed ..."?

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 14]
```diff
@@ -1786,17 +1911,21 @@ Besides the `--name` option, this command also takes the default `--help`, `-f`/
 > runtime-mgr server token [options]
 ----
 
-This command gets server registration token. This token needs to be used to register a new server. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command gets server registration token. This token needs to be used to register a new server.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1916]: "gets a registration token" or "the registration token"

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 15]
```diff
@@ -1836,25 +1967,30 @@ This command does not take any options, except for the default ones: `--help`, `
 > runtime-mgr serverGroup create [options] <name> [serverIds...]
 ----
 
-This command creates a server group with the name passed in `name` using the server Id(s) passed as argument(s) thereafter. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command creates a server group with the name passed in `name` using the server Id(s) passed as argument(s) thereafter.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1972]: "using all IDs"? do we allow constructions like "argument(s)"?

Comment (dukesphere) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1972]: "arguments"

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 16]
```diff
@@ -1887,35 +2027,43 @@ This command does not prompt twice before deleting. If you send a delete instruc
...
 
 === runtime-mgr serverGroup remove server
 
 ----
 > runtime-mgr serverGroup remove server [options] <serverGroupId> <serverId>
 ----
 
-This command removes the removes the server passed in `serverId`, from the server group passed in `serverGroupId`. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command removes the removes the server passed in `serverId`, from the server group passed in `serverGroupId`.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc]:
```suggestion
This command removes the server passed in `serverId` from the server group passed in `serverGroupId`. 
```

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 17]
```diff
@@ -2047,8 +2203,10 @@ Besides the default `--help`, `-f`/`--fields` and `-o`/`--output` options, this
 > runtime-mgr standalone-alert list [options]
 ----
 
-This command lists all alerts for standalone Mules in the current environment. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command lists all alerts for standalone Mules in the current environment.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:2208]: I don't think we allow "Mules." Prob should be "Mule instances".

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 18]
```diff
@@ -2057,27 +2215,36 @@ This command does not take any options, except for the default ones: `--help`, `
 ----
 > cloudhub load-balancer list [options]
 ----
-This command lists all load balancers in your Anypoint Platform. It displays load balancer's name, domain, its state and the Anypoint VPC Id to which the load balancer is bound. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command lists all load balancers in your Anypoint Platform. It displays load balancer's name, domain, its state and the Anypoint VPC Id to which the load balancer is bound.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc]:
```suggestion
This command lists all load balancers in your Anypoint Platform. It displays load balancer's name, domain, its state, and the Anypoint VPC ID to which the load balancer is bound. 
```

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 19]
```diff
@@ -2057,27 +2215,36 @@ This command does not take any options, except for the default ones: `--help`, `
...
 
 === cloudhub load-balancer describe-json
 
 ----
 > cloudhub load-balancer describe [options] <name>
 ----
-This command displays raw JSON response of the load balancer that is specified in `<name>`. +
-You can start typing your load balancer's name and press`tab` for Anypoint Platform CLI to autocomplete it, or you can double tap `tab` for a full list of all the values you can pass. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command displays raw JSON response of the load balancer that is specified in `<name>`.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc]:
```suggestion
This command displays a raw JSON response from the load balancer that is specified in `<name>`. 
```

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 20]
```diff
@@ -2322,15 +2489,19 @@ This command does not take any options, except for the default ones: `--help`, `
...
+// CLI DEFAULTS
+include::partial$cli-default-options.adoc[tag=CLIdefaultOptions]
 
 === cloudhub runtime list
 ----
 > cloudhub runtime list [options]
 ----
-This command lists all supported runtimes. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command lists all supported runtimes.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:2503]: Are we talking about Mule runtime engines? Or some other runtime. Only use "runtimes" alone if Mule is understood from the context.

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 21]
```diff
@@ -2404,8 +2582,10 @@ Besides the default `--help` option, this command also takes the option `--defau
 ----
 > cloudhub vpc business-groups add [options] <vpc> <businessGroups...>
 ----
-This command assigns the Anypoint VPC defined in `<vpc>` to the business group(s) passed as argument(s) thereafter. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command assigns the Anypoint VPC defined in `<vpc>` to the business group(s) passed as argument(s) thereafter.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:2587]: Do we allow constructions like "group(s)" and "argument(s)" ?

Comment (dukesphere) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:2587]: "passed thereafter" works for me. Not sure if we normally use it.

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 22]
```diff
@@ -2506,8 +2694,10 @@ This command is only available for interactive mode.
 ----
 > use environment [options] <name>
 ----
-This command makes active the environment specified in `<name>`. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command makes active the environment specified in `<name>`.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:2699]: "makes active" -> "activates"?

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 23]
```diff
@@ -1049,16 +1110,19 @@ This Command deletes the Design Center project specified in `name`.
...
 
 === designcenter project download
 
 ----
 > designcenter project download [options] <name> <targetDir>
 ----
 
-This Command downloads the Design Center project passed in `name` to your local directory specified in `targetDir`. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This Command downloads the Design Center project passed in `name` to your local directory specified in `targetDir`.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc]:
```suggestion
This command downloads the Design Center project passed in `name` to your local directory specified in `targetDir`.
```

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 7 above)
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc]:
```suggestion
The command returns data such as the application's domain, its status, the last time it was updated, the Mule version, the ZIP file name, the region, monitoring, and workers; as well as `TRUE` or `FALSE` information for persistent queues and static IPs enablement. 
```

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 8 above)
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1477]:
```suggestion
You can start typing your application's name and press `tab` for Anypoint Platform CLI to autocomplete it, or you can double-tap `tab` for a full list of all the values you can pass. 
```

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 24]
```diff
@@ -1722,31 +1837,37 @@ Besides the default `--help`, `-f`/`--fields` and `-o`/`--output` options, this
...
-This command copies the standalone (on-premises) application passed in `source` to the target passed in `target` and the server, server group or cluster ID or Name passed in `targetIdentifier`. +
+This command copies the standalone (on-premises) application passed in `source` to the target passed in `target` and the server, server group or cluster ID or Name passed in `targetIdentifier`. 
+
 Both arguments `source` and `destination` are represented using the format: `<organizationName>:<environmentName>/<appName>`, for example:
 
 ----
 > runtime-mgr standalone-application copy Services:QA/application-1 Development:QA/application-2 123456
 ----
-Copies the application named `application-1` from the QA environment of the _Services_ organization to the QA environment of the `_Development_` organization in the server Id 123456. +
+Copies the application named `application-1` from the QA environment of the _Services_ organization to the QA environment of the `_Development_` organization in the server Id 123456.
```
Comment (dukesphere) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc]:
```suggestion
Copies the application named `application-1` from the QA environment of the _Services_ organization to the QA environment of the `_Development_` organization in the server ID 123456. 
```

**Review by dukesphere — COMMENTED** [review]
some comments. some small, others more impt

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 1 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc]: Yeah, I know. I didn't edit that content - just replaced the boilerplate with the include, which apparently triggered a whole lot of green.

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 2 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1039]: No. That's what the CLI source says.

Comment (hannanelson) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1039]:
```suggestion
This command adds a dedicated load balancer configuration specified by `<dlbUrl>` to Anypoint DataGraph.
The `dlbUrl` is a valid URL that includes the DLB domain and the mapping rule `inputUri`.

```

Comment (dukesphere) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1039]: I wonder if the descriptions in the source are mixed up bc of what I see below/next.

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 4 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1392]: That's the way Kim prefers it - that you don't repeat the subject.

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 5 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1451]: Yes, but I'm not editing that text right now.

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 6 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1463]: Yes, but I'm leaving rewriting until next time.

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 7 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc]: Yep!

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 8 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1477]: Not updating everything just now.

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 9 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1664]: Yep!

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 10 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1721]: Nope!

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 11 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1750]: Yep!

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 13 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1864]: Nope - it should be "name"

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 14 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1916]: Yep!

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 15 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1972]: Ditto

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 17 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:2208]: Yep!

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc (hunk 21 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-platform-cli-commands.adoc:2587]: Nope to the first.

**Review by dukesphere — APPROVED** [review]

**Pull request merged**